packages/react/src/components/SearchField/index.tsx

```diff
@@ -101,8 +102,7 @@ const SearchField = forwardRef<HTMLInputElement, SearchFieldProps>(
             onChange={handleChange}
             placeholder={placeholder}
             ref={ref}
-            {...otherProps}
-            className={classNames(otherProps.className, 'Field__text-input')}
```

Comment:
This was likely a mistake in the initial implementation, `TextFieldWrapper` should appropriately handle any styles associated with input and this component should not have this class.